api/lib/domain/usecases/update-certification-center-data-protection-officer-information.js

```diff
@@ -0,0 +1,13 @@
+module.exports = async function updateCertificationCenterDataProtectionOfficerInformation({
+  dataProtectionOfficer,
+  dataProtectionOfficerRepository,
+}) {
```

Comment:
Le nom de la fonction est ici inutile, alors autant ne pas le mettre :

```suggestion
module.exports = async function({ dataProtectionOfficer, dataProtectionOfficerRepository }) {
```

Reply:
Je dirais que oui parce que le nom se trouve déjà sur le fichier, et non pour le debugging ça peut parfois aider. Mais je ne pourrais pas le modifier sans modifier tous les usecases de l'API.